frontend/src/pages/cans/detail/BudgetSummary.jsx

```diff
@@ -0,0 +1,123 @@
+import { useSelector, useDispatch } from "react-redux";
+import { getCanFiscalYearByCan } from "./getCanFiscalYear";
+import { useEffect, useState } from "react";
+import Select from "react-select";
+import { useParams } from "react-router-dom";
+
+const BudgetSummary = () => {
+    const dispatch = useDispatch();
+    const can_fiscal_year = useSelector((state) => state.canFiscalYearDetail.canFiscalYear);
+    const urlPathParams = useParams();
+    const [selectedValue, setSelectedValue] = useState(0);
+    const handleChange = (e) => {
+        setSelectedValue(e.value);
+        // console.log(cfy[0].fiscal_year);
+    };
```

Comment:
We shouldn't need to use React's `useState` because we're using Redux for state management.

So, in `handleChange`, you would do...

```javascript
dispatch(setFiscalYear(e.value));
```

That would require a `setFiscalYear` reducer and its own state in `initialState`.  Lastly, you'd have to add a `useSelector` for the fiscal year, and update the `useEffect` dependency list.